docs/_content/ontwikkelaars/algemeen/archiveren.md

```diff
@@ -49,15 +53,100 @@ gerelateerde entiteiten:
+
+*Voorbeeld*
+
+Als `ResultaatType.BrondatumArchiefprocedure.Afleidingswijze` wordt ingesteld op `eigenschap` moet het ZTC valideren dat het betreffende `ZaakType` een `Eigenschap` heeft met als `naam`, de waarde die staat in `ResultaatType.BrondatumArchiefprocedure.datumkenmerk`.
+
+### Berekenen van de `Zaak.Archiefactiedatum`
+
+1. Bepaal de *brondatum* van de `Zaak` door de `ResultaatType.BrondatumArchiefprocedure.Afleidingswijze` te raadplegen:
+
+   Afleidingswijze | Waarde van *brondatum*
+   --- | ---
+   `afgehandeld` | `Zaak.Einddatum`
+   `gerelateerde zaak` | *TODO: Wat is dit precies?* De hoogste datum van van alle `Zaak.GerelateerdeZaak.Einddatum` of `Zaak.Einddatum` ([#776](https://github.com/VNG-Realisatie/gemma-zaken/issues/776))
+   `hoofdzaak` | `Zaak.HoofdZaak.Einddatum`
+   `ingangsdatum besluit` | *TODO* `Zaak.Besluit.Ingangsdatum` ([#775](https://github.com/VNG-Realisatie/gemma-zaken/issues/775))
+   `vervaldatum besluit` | *TODO* `Zaak.Besluit.Vervaldatum` ([#775](https://github.com/VNG-Realisatie/gemma-zaken/issues/775))
+   `ander datumkenmerk` | *TODO: Dubbelcheck* Niet te bepalen.
+   `eigenschap` | De waarde van de `Zaak.Eigenschap` met de `naam` die overeenkomt met de waarde uit `ResultaatType.BrondatumArchiefprocedure.Datumkenmerk`
+   `termijn` | `Zaak.Einddatum` + `ResultaatType.BrondatumArchiefprocedure.Procestermijn`
+   `zaakobject` | De waarde van het attribuut op `Zaak.ZaakObject.Object`, van type `ResultaatType.BrondatumArchiefprocedure.Objecttype`, waarvan de `naam` van het attribuut overeenkomt met de waarde uit `ResultaatType.BrondatumArchiefprocedure.Datumkenmerk`
+
+2. Als de *brondatum* is bepaald:
+
+   `Zaak.Archiefactiedatum` = *brondatum* + `Zaak.Resultaat.ResultaatType.Archiefactietermijn`
+
+User Story: [#345](https://github.com/VNG-Realisatie/gemma-zaken/issues/345)
+
+#### Foutsituaties bij het berekenen van *brondatum*
+
+In sommige situaties kan de *brondatum* niet worden bepaald. Dit is niet altijd een foutsitatie. In het algemeen moet een fout optreden als de configuratie in het ZTC niet overeenkomt met de inrichting van de `Zaak` of gerelateerde gegevens, of als er ongeldige waardes zijn.
+
+*Voorbeeld*
+
+Als de afleidingswijze een `eigenschap` betreft en de `Zaak` heeft zo'n `eigenschap` niet, dan treed een fout op. Ook als de `eigenschap` wel bestaat maar de waarde is geen geldige datum, dan treed een fout op. Echter, als de `eigenschap` bestaat en de waarde is leeg, dan kan de *brondatum* niet worden bepaald en blijft de `Zaak.archiefactiedatum` leeg
+
+### Wanneer wordt de `Zaak.Archiefactiedatum` berekend?
+
+De `Zaak.Archiefactiedatum` wordt berekend als aan de volgende voorwaarden wordt voldaan:
+
+1) Er wordt een `Resultaat` voor de `Zaak` aangemaakt of gewijzigd; hiermee wordt het `ResultaatType` bekend,
+2) De *brondatum* kan worden berekend,
+3) De `ResultaatType.archiefactietermijn` van het `Zaak.Resultaat` is valide.
+
```

Comment:
Dit geldt als de archiefnominatie vernietigen is. En wat doe je als de archiefnominatie blijvend bewaren is? Dan is archiefactietermijn 20 jaar denk ik.

Reply:
Nu nog wel ja :)